### ISSUE TITLE
Possible to ignore a specific line?

### ISSUE THREAD

**Issue by corysimmons** [stylelint/stylelint#714]
It'd be nice if I could ignore specific lines a la a comment or something.

```css
.foo {
  color: red; /* stylelint-ignore */
}
```

Is anything like this possible with stylelint or is this a feature that anyone else is interested in?

**Comment by davidtheclark** [comment]
Currently you should be able to do this with:

```css
.foo {
  /* stylelint-disable */
  color: red; 
  /* stylelint-enable */
}
```

Allowing `/* stylelint-ignore */` at the end of a line to ignore that line would be a new feature.

**Comment by corysimmons** [comment]
What are your thoughts about it? After messing around  a bit more I realized I didn't really need half the ignores.

I don't think this is mission critical or anything -- just sugar.

**Comment by davidtheclark** [comment]
Yeah since it's already possible I'd say this is low priority unless someone comes along and wants it enough to build it.

**Comment by davidtheclark** [comment]
Actually, since this is already possible I think there's no need to add more to the codebase for a little sugar.

**Comment by esseb** [comment]
For other people coming to this issue from Google in the future, this was implemented in https://github.com/stylelint/stylelint/issues/763:

```css
#id { /* stylelint-disable-line */
  color: pink !important; /* stylelint-disable-line declaration-no-important */
}
```

See https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md

**Comment by evilebottnawi** [comment]
@esseb i can help us, resolved https://github.com/stylelint/stylelint/issues/1614 and use
```css
#id {
  /* stylelint-disable-next-line declaration-no-important */
  color: pink !important;
}
```
:smile:

**Comment by Spixmaster** [comment]
It does not work for me.

```css
[data-comment-depth]
{
    /* stylelint-disable-next-line no-unknown-custom-properties */
    border-inline-start: 0.15rem solid var(--bs-secondary-border-subtle);
    padding-inline-start: 1rem;
}   
```

**Comment by Mouvedia** [comment]
@Spixmaster [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIDaUCGAXVBaSBbXMAdulrAA7oAWAugDqHD0AELTA9AFRMDO6AngBsYAgJbFSI7qgBGQrIRgAPEqIVNCELAFdCAaw0B3Qji28IuLGQBOEMjCvoRMbkw5tmraRCuwrWMaowWLyoDvBMAAwAdACMAKxWMLg8EKJQTABuoQAUWFjS3MEwkIRoVnz53r7BWtLoQgCUANweLGSoUFBiAOb+hIHBmGFMMYm4LYQAvqwgADQgAGYiQgByqAQIIErrZEJRYNzcc+AQhEvdm4yELLQgVlpC3LfhV6xMtxraeobGYKbo5ksNjsDicTwQTHQ9xgHkm9EmxxK5wAYt5cBhNgArbinY7kI6IV7vEC8QTCMToZ7EgQYZyUuYeW6koSqEhIkS9EKlUJQKm3GnoOm3eEgSZAA)

**Comment by Spixmaster** [comment]
@Mouvedia Thank you very much! I found the issue. It was my configuration file.

```yaml
configurationComment:
```

Commenting the setting out solved the issue.
